spack-environments/care/spack.yaml

```diff
@@ -0,0 +1,79 @@
+##############################################################################
+# Copyright (c) 2020-2021, Lawrence Livermore National Security, LLC and
+# RADIUSS Stack project contributors. See the LICENSE file for details.
+#
+# SPDX-License-Identifier: MIT
+##############################################################################
+
+spack:
+  #[general--]
```

Comment:
Those comments are delimiters for documentation, no need to keep them outside of the original file.